### DISCUSSION TITLE
🔥🔥🔥 NetBox DNS has moved to a new home 🔥🔥🔥

### DISCUSSION THREAD

**Discussion by peteeckel** [auroraresearchlab/netbox-dns#309] [General]
Sorry for repeating this ... NetBox DNS is not here anymore.

Please do **not** star this repository anymore. The high number of stars leads many users to use this repo and the wrong (i.e. outdated and unsupported) PyPI module, and this causes unnecessary frustration on their and unnecessary issues and explanations on my side.

**If possible, plesse remove your stars for this - the old, unsupported and, since NetBox 3.5, broken - repository and move them over to [the new, supported fork](https://github.com/peteeckel/netbox-plugin-dns).** The high number of stars still prompts users to try this code and fail, and then move over to the new code and fail again (because the old one hasn't been removed properly before).

I will continue to watch this repository for issues, but I can no longer make any fixes or releases as the maintainers have abandoned the project.

Sorry for the inconvenience. I would have preferred the transition to have been more smoothly, or no transition at all, but here we are.

**Comment by peteeckel** [comment]
There are still new stars appearing for this repo ...

Again, the repository is dead and the development of NetBox DNS has been moved. The maintainers are gone, I can't do any updates and development has taken place elsewhere. By starring this old repo it is still gaining weight in searches and other people will be attracted to it - though there is nothing happening here anymore, for about a year now.

@bertogravscale
@nathen418
@viehoang68
@luciendar81
@jvalente-salemstate
@soundtech2003
@roycohen2013
@roycohen2013
@pertsevds
@pcslusa
@n0nvme
@carleski
@loveablecabbage
@skorvek
@KUBAllen
@RENANZG
@escrimaglia
@DMel74
@jscmenezes
@etdi
@Jaharmi
@mvandermeulen
@qdrddr
@brotherdust
@fabio-anzola

(to name just the last few) - could you all please check if you really wanted to star this repo? If it happened by mistake, please remove the star from this old repository so others don't get attracted to this one and wonder why nothing happens anymore. The project itself is pretty much alive, just not here ...

**Comment by escrimaglia** [comment]
Hi Peter,

Done!

Where the development of NetBox DNS is now?

br

De: Peter Eckel ***@***.***>
Fecha: miércoles, 27 de marzo de 2024, 17:49
Para: auroraresearchlab/netbox-dns ***@***.***>
CC: Ed Scrimaglia ***@***.***>, Mention ***@***.***>
Asunto: Re: [auroraresearchlab/netbox-dns] 🔥🔥🔥 NetBox DNS has moved to a new home 🔥🔥🔥 (Discussion #309)

There are still new stars appearing for this repo ...

Again, the repository is dead and the development of NetBox DNS was moved. The maintainers are gone, I can't do any updates and development has taken place elsewhere. By starring this old repo it is still gaining weight in searches and other people will be attracted to it - though there is nothing happening here anymore, for about a year now.

@bertogravscale<https://github.com/bertogravscale>
@nathen418<https://github.com/nathen418>
@viehoang68<https://github.com/viehoang68>
@luciendar81<https://github.com/luciendar81>
@jvalente-salemstate<https://github.com/jvalente-salemstate>
@soundtech2003<https://github.com/soundtech2003>
@roycohen2013<https://github.com/roycohen2013>
@roycohen2013<https://github.com/roycohen2013>
@pertsevds<https://github.com/pertsevds>
@pcslusa<https://github.com/pcslusa>
@n0nvme<https://github.com/n0nvme>
@carleski<https://github.com/carleski>
@loveablecabbage<https://github.com/loveablecabbage>
@skorvek<https://github.com/skorvek>
@KUBAllen<https://github.com/KUBAllen>
@RENANZG<https://github.com/RENANZG>
@escrimaglia<https://github.com/escrimaglia>
@DMel74<https://github.com/DMel74>
@jscmenezes<https://github.com/jscmenezes>
@etdi<https://github.com/etdi>
@Jaharmi<https://github.com/Jaharmi>
@mvandermeulen<https://github.com/mvandermeulen>
@qdrddr<https://github.com/qdrddr>
@brotherdust<https://github.com/brotherdust>
@fabio-anzola<https://github.com/fabio-anzola>

(to name just the last few) - could you all please check if you really wanted to star this repo? If it happened by mistake, please remove the star from this old repository so others don't get attracted to this one and wonder why nothing happens anymore. The project itself is pretty much alive, just not here ...

**Reply by peteeckel** [reply]
Thanks! The new repo is here: https://github.com/peteeckel/netbox-plugin-dns

**Comment by escrimaglia** [comment]
Thanks! Sent from my iPhone, apologize for the brevity and typos. El 27 mar 2024, a la(s) 18:43, Peter Eckel ***@***.***> escribió:﻿
Thanks! The new repo is here: https://github.com/peteeckel/netbox-plugin-dns

**Comment by peteeckel** [comment]
@FabbyoMalta
@TheLastFireBender
@bertogravscale

See above ... you're at the wrong place, really ... there's nothing to see here :-)

**Comment by peteeckel** [comment]
It's really exhausting ... 

@Matt-Deacalion
@tcx9871
@ransbymich

**Please** read this discussion. **Please** remove your star from this repository. 

By starring it, you push it further up in the search rank. And that attracts more users to it. And they star it ... and so on.

auroraresearchlab/netbox-dns is DEAD. Abandoned. Finished. Unsupported. Nothing happens anymore in this repo. The new one is at [peteeckel/netbox-plugin-dns](https://github.com/peteeckel/netbox-plugin-dns), and it's the repository for the supported NetBox DNS plugin certified by NetBox Labs.

Upvoting the dead old repository just makes it harder to find the correct one, where releases are announced, errors are fixed and potentially important information is published. And it creates unnecessary work for me, which damages development of NetBox DNS.

Thank you.
